Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2-slim
+RUN groupadd user && useradd --home /code --disabled-password --gecos "" -g user user
+
+COPY "feedfetcher.py","rssfeed.py","settings.py.docker" /code

```

Comment:
This is an invalid command:
`ERROR: Service 'mattermost-rss' failed to build: lstat feedfetcher.py,rssfeed.py,settings.py.docker: no such file or directory`.
Even if this would work, you're missing the `requirements.txt` which is used in the next step.

My suggestion: `COPY . /code`